test/xcode-specs.js

```diff
@@ -83,4 +83,10 @@ describe('xcode @skip-linux', () => {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
+  it('should get a list of iOS devices', async() => {
+    let devices = await xcode.getConnectedDevices();
+    should.exist(devices);
```

Comment:
Sorry I thought it was @Jonahss PR, don't worry about it, I'll refactor that later.